db/hybrid_logical_vector.go

```diff
@@ -114,24 +71,33 @@ func (hlv *HybridLogicalVector) ExtractCurrentVersionFromHLV() *Version {
 // PersistedHybridLogicalVector is the marshalled format of HybridLogicalVector.
 // This representation needs to be kept in sync with XDCR.
 type HybridLogicalVector struct {
-	CurrentVersionCAS string            `json:"cvCas,omitempty"`     // current version cas (or cvCAS) stores the current CAS in little endian hex format at the time of replication
-	ImportCAS         string            `json:"importCAS,omitempty"` // Set when an import modifies the document CAS but preserves the HLV (import of a version replicated by XDCR)
-	SourceID          string            `json:"src"`                 // source bucket uuid in (base64 encoded format) of where this entry originated from
-	Version           string            `json:"ver"`                 // current cas in little endian hex format of the current version on the version vector
-	MergeVersions     map[string]string `json:"mv,omitempty"`        // map of merge versions for fast efficient lookup
-	PreviousVersions  map[string]string `json:"pv,omitempty"`        // map of previous versions for fast efficient lookup
+	CurrentVersionCAS uint64      // current version cas (or cvCAS) stores the current CAS in little endian hex format at the time of replication
+	ImportCAS         uint64      // Set when an import modifies the document CAS but preserves the HLV (import of a version replicated by XDCR)
+	SourceID          string      // source bucket uuid in (base64 encoded format) of where this entry originated from
+	Version           uint64      // current cas in little endian hex format of the current version on the version vector
+	MergeVersions     HLVVersions // map of merge versions for fast efficient lookup
+	PreviousVersions  HLVVersions // map of previous versions for fast efficient lookup
+}
+
+type BucketVector struct {
```

Comment:
I don't love having a separate struct just for persistence, but we can revisit in the context of the PV delta support work.

Reply:
Yeah will have this sorted in the deltas ticket, it wasn't immediately clear to me the best approach so thought this will do to just unblock cbl for now.